src/javascript/projenrc.ts

```diff
@@ -47,6 +49,11 @@ export class Projenrc extends ProjenrcFile {
         "import/no-extraneous-dependencies": "off",
       },
     });
+
+    if (this.project instanceof NodeProject) {
+      const biome = Biome.of(this.project);
+      biome?.addLintPattern(this.filePath);
+    }
```

Comment:
let the presence of the Biome component be enough indication. I.e. remove the if.
```suggestion
    const biome = Biome.of(this.project);
    biome?.addLintPattern(this.filePath);
```